docs/bgp-policy.md

```diff
@@ -93,6 +107,11 @@ provider to avoid issues caused by private ASN usage.
 The `listenPort` field specifies the port on which the BGP process listens. The default value is 179. The valid port
 range is `1-65535`.
 
+### Confederation
+
+The `Confederation` field specifies that the BGP process operates within a confederation identified by ASN `65000`,
+with other member ASNs `64513` and `64514`.
```

Comment:
This is the only field for which the description refers to specific values used in the example, so that's inconsistent. I would recommend just having a generic description here, and adding a link to the more comprehensive confederation example which comes later in the document.

Reply:
Done